app/wizards/placements/edit_placement_wizard.rb

```diff
@@ -0,0 +1,46 @@
+module Placements
+  class EditPlacementWizard < BaseWizard
+    attr_reader :school, :placement
+
+    def initialize(school:, params:, placement:, session:, current_step: nil)
+      @school = school
+      @placement = placement
+      super(session:, params:, current_step:)
+    end
+
+    def define_steps
+      # Define the wizard steps here
+      add_step(ProviderStep) if current_step == :provider
+      add_step(AddPlacementWizard::YearGroupStep) if current_step == :year_group
+      add_step(AddPlacementWizard::MentorsStep) if current_step == :mentors
+    end
```

Comment:
I'm gonna throw another suggestion into the mix, just because I'm annoying like that.

```suggestion
    def define_steps
      # Define the wizard steps here
      possible_steps = {
        provider: ProviderStep,
        year_group: AddPlacementWizard::YearGroupStep,
        mentors: AddPlacementWizard::MentorsStep,
      }

      add_step(possible_steps.fetch(current_step))
    end
```

Although to be fair, I'm happy with whatever here. It's all a matter of opinion & preference.

Reply:
Agreed to keep as is for now, can re-visit in the future if a new pattern emerges. This is consistent with other step definitions